mhs/outbound/main.py

```diff
@@ -140,7 +140,7 @@ def main():
                                                   ca_certs=secrets.get_secret_config('CA_CERTS'))
     max_retries = int(config.get_config('OUTBOUND_TRANSMISSION_MAX_RETRIES', default="3"))
     retry_delay = int(config.get_config('OUTBOUND_TRANSMISSION_RETRY_DELAY', default="100"))
-    validate_cert = bool(config.get_config('OUTBOUND_VALIDATE_CERTIFICATE', default="True"))
+    validate_cert = config.get_config('OUTBOUND_VALIDATE_CERTIFICATE', default=str(True)).lower() == str(True).lower()
```

Comment:
If env var is set then `config.get_config()` will return String. I think that isn't the way it should be

Reply:
ok, as I read it I saw `default=str(True).lower()`. Looks like it's easy to misread it. Please split that into two lines